tests/src/readytorun/mainv2.csproj

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I do not think we should be compiling mainv2. The key thing for these tests to verify is that main.ni.exe compiled against test.dll v1 works well with test.dll v2. In other words, that the lower layer can be changed without breaking native code in the higher layer.